source/extensions/filters/http/aws_lambda/config.cc

```diff
@@ -43,6 +43,8 @@ AWSLambdaConfigImpl::AWSLambdaConfigImpl(
     : stats_(generateStats(stats_prefix, scope)), api_(api),
       file_watcher_(dispatcher.createFilesystemWatcher()), tls_(tls) {
 
+  propagateOriginalRouting_ = protoconfig.propagate_original_routing().value();
```

Comment:
`propagate_original_routing_` and should be set above aka:
`propagate_original_routing_(protoconfig.propagate_original_routing().value())`

Reply:
oh yes should use the initialization list, agreed

Reply:
Question, I had tried this previously and I took another shot at rearranging things again but I have the following issue.

This is the Impl of AwsConfig rather than the basic awsconfig. If there is a good way for the check in aws_lambda_filter using the shared abstract AwsConfig to be cast to a AwsConfigImpl then we are good otherwise I dont see how I can use the Impl's initialization list. 

Ill see if I cant find some docs for how c++ works with regards to this.

Reply:
if you don't find anything helpful feel free to DM me, we can screen share and go over it together. should be very quick

Reply:
Alright so after call with kevin I have now moved this as it was originally being set in a incorrect space. Namely we couldn't inline it here as it was on the virtual class rather than the impl. 
After talking it through it seemed that it was better to move it to the filter instantiation as we had access to the proto rather than adding another virtual method on the config as a getter for the value that would be placed on the impl.

I have now pushed this update.